src/reifyhealth/specmonstah/core.cljc

```diff
@@ -675,7 +676,8 @@
                               #{ref-ent} ref-ent)}))))
 
 (s/fdef ent-relations
-  :args (s/tuple ::db ::ent-name)
+  :args (s/or :n-1 (s/cat :db ::db)
+              :n-2 (s/cat :db ::db :ent-name ::ent-name))
```

Comment:
I think there shouldn't be a 2-arity spec here, since `ent-relations` always has two params.